### ISSUE TITLE
Suggestion: "short bus"

### ISSUE THREAD

**Issue by malchata** [tatianamac/selfdefined#433]
One phrase I've heard in colloquial conversation and in some media is "short bus", which is offered as a synonym for the r-word. Sources on this word (CW: these sources may not abbreviate the offending term to "r-word"):

- [Wikipedia entry](https://en.wikipedia.org/wiki/Short_Bus).
- [Online Slang Dictionary entry](http://onlineslangdictionary.com/meaning-definition-of/ride-the-short-bus).

I think if we're going to highlight the r-word (and one would be right to do so), we should offer this as a synonym in some place. It's a term of derision people use to sidestep actually _saying_ the r-word, but has similar, hurtful outcomes.

**Comment by tatianamac** [comment]
Agreed, @malchata ! Would you feel comfortable taking a first pass at a definition with my support?

**Comment by malchata** [comment]
Sure! I just need to finish up some other stuff before I get to it. Might be next week. Cool?
